safepickle/safepickle.py

```diff
@@ -1,6 +1,7 @@
 import json
 
 from .encoding import Encoder, Decoder
+from pickle import PickleError, PicklingError, UnpicklingError
```

Comment:
Do we use `PickleError`? When is that used in the standard pickle module? It might be useful to raise that error on unknown exceptions (`except Exception`) but raise `PicklingError` when we catch known exceptions (`except ValueError`)